slack_rules/slack_passthrough_anomaly.py

```diff
@@ -0,0 +1,10 @@
+from panther_base_helpers import slack_alert_context
+
+
+def rule(event):
+    return event.get("action") == "anomaly"
+
+
+def alert_context(event):
+    # TODO: Add more details to context
```

Comment:
nit

Reply:
This is b/c we don't have this data internally and can't find any examples online -- there's more to be added to the context but we don't have access to what that data looks like